tests/coalaTest.py

```diff
@@ -47,22 +47,22 @@ def test_show_all_bears(self):
             retval, output = execute_coala(coala.main, 'coala', '-B')
             self.assertEqual(retval, 0)
             # 6 bears plus 1 line holding the closing colour escape sequence
-            self.assertEqual(len(output.strip().splitlines()), 7)
+            self.assertEqual(len(output.strip().splitlines()), 8)
 
     def test_show_language_bears(self):
         with bear_test_module():
             retval, output = execute_coala(
                 coala.main, 'coala', '-B', '-l', 'java')
             self.assertEqual(retval, 0)
             # 2 bears plus 1 line holding the closing colour escape sequence
```

Comment:
you gotta update those comments, you can't just change the tests and expect me to believe this is right :(